packages/grafana-ui/src/components/Table/TableCellInspector.tsx

```diff
@@ -28,7 +28,7 @@ export function TableCellInspector({ value, onDismiss, mode }: TableCellInspecto
     if (trimmedValue[0] === '{' || trimmedValue[0] === '[' || mode === 'code') {
       try {
         value = JSON.parse(value);
-        displayValue = JSON.stringify(value, null, '');
+        displayValue = JSON.stringify(value, null, ' ');
```

Comment:
If I'm understanding this correctly, this try/catch is a way to attempt JSON parsing without disrupting the flow of the component if the parsing fails, since if an error occurs during parsing, the code silently ignores it and continues execution. If that's correct, do we at least want to log the error in the console? Something like:
```
try {
  value = JSON.parse(value);
  displayValue = JSON.stringify(value, null, ' ');
} catch (error) {
  logger.debug('Failed to parse JSON in TableCellInspector', { error: error.message, value });
}
```

Reply:
Thanks for the suggestion Jev! I wouldn't normally log things to the console but seems reasonable in this case if someone wants to dig into why their JSON output might not be formatting the way they want. I've added it 😄